apps/docs/src/content/reference/extras/image.mdx

```diff
@@ -0,0 +1,56 @@
+---
+{

```

Comment:
I would like us to not continue the `{ }` brackets within our components. It was giving me troubles when I was mucking around with starlight

Reply:
Updated! The non-brackets version is what astro recommends, correct? If so, is there any progress on moving all of our docs to using that format? I'm happy to help in any way I can to help move that along so that we're using the correct format everywhere.

Reply:
Yeah, I couldn't find the brackets version in Astro docs when I looked into this. Almost there on the progress, have a look over #880 when you get some time